packages/vue/src/components/atoms/SfIcon/SfIcon.vue

```diff
@@ -29,6 +29,13 @@
         />
       </svg>
     </slot>
+    <slot name="badge" v-bind="{ badgeLabel, hasBadge }">
```

Comment:
please remove this. We already concluded that we have this removed from `SfIcon` since it's not the correct behavior of an Atom component.

Reply:
Yes of course I forgot to remove it after changes, sorry.